mypy/fscache.py

```diff
@@ -121,20 +121,20 @@ def exists(self, path: str) -> bool:
         return True
 
 
+# TODO: Merge FileSystemMetaCache back into this?
```

Comment:
Would it be much work to do that now? Why keep them separate?

Reply:
Not much work, but I didn't want to clutter up this PR. I'll merge them in another PR after this?